data/brands/shop/electronics.json

```diff
@@ -757,9 +757,7 @@
     {
       "displayName": "Power",
       "id": "power-9765c4",
-      "locationSet": {
-        "include": ["dk", "fi", "no", "se"]
-      },
+      "locationSet": {"include": ["dk", "fi", "no", "se"]},
```

Comment:
Notice:  You don't need to let this in 1 line, program will auto split if it is too long.